src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/entities/AsylumCaseFieldDefinition.java

```diff
@@ -42,7 +43,10 @@
         "appellantFamilyName", new TypeReference<String>(){}),
 
     APPELLANT_NATIONALITIES(
-            "appellantNationalities", new TypeReference<List<String>>(){}),
+            "appellantNationalities", new TypeReference<List<IdValue<NationalityFieldValue>>>(){}),
```

Comment:
this might be a breaking change, can we somehow avoid it? Did you test if existing cases work after the type change?

Reply:
Yes. I had a meeting with Brad, as it was added only for StatelessHandler. Did the analysis and tested RIA-2010 again in functional and E2E test. I have asked Brad to review this PR too. Updated story requesting Srihari to test Stateless flow in E2E.
I could not avoid making the change as I have to read the values from CCD. It cannot read as List<String>. It worked for Stateless as they were only writing to Asylumcase and not using objectMapper